pkg/operator/target_config_reconciler.go

```diff
@@ -1008,6 +1204,11 @@ func (c *TargetConfigReconciler) manageConfigMap(descheduler *deschedulerv1.Kube
 		}
 	}
 
+	if descheduler.Spec.ProfileCustomizations != nil && descheduler.Spec.ProfileCustomizations.DevKubevirtSchedulable {
```

Comment:
This needs to go under `relieveAndMigrateProfile` as it's relevant only to the new profile. Alternatively, you could introduce a separate `DevNodeSelector` tuning option for all plugins/profiles. Either as a string (to allow any node selector) or a predefined list of options (e.g. master nodes, worker nodes, kubevirtschedulable nodes`. Given we have not had any request from customers for this functionality it's better to have it scoped only for the profile for now. Also, would it make more sense to have this configuration by default without any tuning options? Are there are kubevirt use cases where it's better to also include nodes which does not have the label?